civihr_employee_portal/src/View/AbstractView.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\View;
+
+abstract class AbstractView {
```

Comment:
Please add a description explaining why/when one would want to extend this class